roles/libvirt_rhel_vm/molecule/shared/prepare.yml

```diff
@@ -0,0 +1,44 @@
+- name: register RHSM
+  hosts: all
+  pre_tasks:
+    - name: install python-apt where necessary
+      pip:
+        # yamllint disable-line rule:line-length
+        name: git+https://git.launchpad.net/ubuntu/+source/python-apt@applied/ubuntu/bionic-updates
+        state: present
+      when: ansible_pkg_mgr == "apt"
+
+    - name: download image file
+      get_url:
+        dest: "~/"
+        # yamllint disable-line rule:line-length
+        url: https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2.xz
+
+    - name: extract file
```

Comment:
Looks like the _unarchive_ module won't work here due to this:

> Does not handle .gz files, .bz2 files or .xz files that do not contain a .tar archive.

Perhaps add a comment explaining why we have to use the _command_ module.

Reply:
Comment added